### PR TITLE
add embroider compatibility testing

### PR DESCRIPTION
**Pull request by SkoebaSteve**
In this PR, we’re introducing two new Ember Try scenario to prepare for Embroider world.
They are for now allowed to fail

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
+    "@embroider/test-setup": "^1.8.3",
```
Comment (dbendaou) [inline package.json:26]: should be in devDependencies :)

Comment (SkoebaSteve) [reply package.json]: Done

**Comment by SkoebaSteve** [comment]
embroider test setup does not support node 10, need to update to v12 to unblock

**Comment by SkoebaSteve** [comment]
@dependabot rebase

**Comment by herzzanu** [comment]
Closing in favour of https://github.com/adopted-ember-addons/ember-launch-darkly/pull/398 It has already been covered by the ember upgrade.

**Pull request closed without merging**